app/javascript/vue/shared/select_dropdown.vue

```diff
@@ -124,7 +125,8 @@ export default {
     searchable: { type: Boolean, default: false },
     clearable: { type: Boolean, default: false },
     tagsView: { type: Boolean, default: false },
-    urlParams: { type: Object, default: () => ({}) }
+    urlParams: { type: Object, default: () => ({}) },
+    e2eValue: { type: String }
```

Comment:
You can define default value here, like `default: ""` and then you don't need `e2eValue ? e2eValue : null`, like you have in previous file